packages/contracts-bedrock/go-fuzz/README.md

```diff
@@ -0,0 +1,32 @@
+# `go-fuzz`
```

Comment:
I think this package / binary should be renamed.

Reply:
Good point- sorry that this feedback got lost, there was some confusion on whether or not @smartcontracts' trie changes were going to be merged and this PR has been closed/reopened a few times.

Any suggestions? Maybe one of:
* `go-ctb-fuzz`
* `go-ctb-fuzzer`
* `ctb-fuzz`

Reply:
I lean to something along the lines of `ctb-test-case-generator`. There's no reason to include the word "go" in this. `ctb-fuzz` is okayish, but I don't actually think of this code as a fuzzer.

Reply:
Name changed to `ctb-test-case-generator`